.golangci.yaml

```
@@ -24,6 +24,9 @@ linters:
     - staticcheck
     - ineffassign
     - bodyclose
+    #- ireturn
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)